### DISCUSSION TITLE
Tracking the full path (https://website.com/?ref=xxx) using router?

### DISCUSSION THREAD

**Discussion by notflip** [MatteoGabriele/vue-gtag#527] [General]
I'm wondering if it's possible to track the full path using the vue-router?
For now in Analytics I only see the name of the path, "Index", instead of the real full path as shown in my title, is this possible?

I'm using the following config in Nuxt 3

```
import VueGtag from 'vue-gtag'

export default defineNuxtPlugin((nuxtApp) => {
    nuxtApp.vueApp.use(VueGtag, {
        config: {
            id: 'G-XXXXX',
        },
    }, nuxtApp.$router)
})
```

**Comment by MatteoGabriele** [comment]
you can use the `pageTrackerUseFullPath` option
